superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -26,13 +26,14 @@ def __init__(self, name, parameters=''):
         self._bin_name = 'fio'
 
         self.__io_patterns = ['seq', 'rand']
-        self.__io_types = ['read', 'write']
+        self.__io_types = ['read', 'write', 'rw']
```

Comment:
Shall we change `rw` to `readwrite`

Reply:
rw here is to align with FIO configuration. FIO recognizes rw, randrw and readwrite, but not randreadwrite.

Reply:
It's interface for SuperBench, and we can translate it internally. @guoshzhao and @abuccts what's your thought for this?

Reply:
Changed to `readwrite` to align with our existing style. I prefer not to introduce both `rw` and `readwrite` here because it's not a must, and it might mislead user to think of them as two different functions.